### PR TITLE
Tutorial salesdemo

### PR DESCRIPTION
**Pull request by amberarrow**
Converted to markdown

### CODE REVIEW

File: docs/tutorials/salesdemo-c1.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+Getting to know DataTorrent RTS
```
Comment (tweise) [inline docs/tutorials/salesdemo-c1.md]: I don't think this should be part of the sales demo. It is a general section that should be moved elsewhere.

**Comment by tweise** [comment]
There are a few sections with content that is not specific to the demo and should be moved out of it. It probably makes sense to have it included into a standalone document (and maybe that's where it came from), but not within this documentation suite.

Other than that, great guide!

**Comment by amberarrow** [comment]
Yes, I noticed that too -- they were in the original doc and I left it alone to get it out quickly. I'll pull them out.

**Comment by amberarrow** [comment]
Removed first two chapters, renamed files to salesdimensions-c?.md, squashed.

File: docs/tutorials/salesdimensions-c1.md [hunk 2]
```diff
@@ -0,0 +1,636 @@
+Building the Sales Dimension application in JAVA
+===
+The Sales Dimensions application demonstrates multiple
+features of the DataTorrent RTS platform, including the ability to
+transform, analyze, and take actions on data in real time. The
+application also demonstrates how DataTorrent RTS can be used to build
```
Comment (gauravgopi123) [inline docs/tutorials/salesdimensions-c1.md]: Can we break these features into List?

Comment (amberarrow) [inline docs/tutorials/salesdimensions-c1.md]: done

**Pull request merged**